library/src/main/java/com/novoda/downloadmanager/NotificationCustomizer.java

```diff
@@ -7,6 +7,13 @@
 
```

Comment:
This class should have a description and link to `DownloadManagerBuilder` method using it